woocommerce-gutenberg-products-block.php

```diff
@@ -293,27 +293,3 @@ function woocommerce_blocks_plugin_outdated_notice() {
  * @param bool $enabled Current filter value.
  * @return bool True if _also_ the `WP_HTML_Tag_Processor` class was found.
  */
```

Comment:
This comment should be removed, shouldn't it? 👀

Reply:
I see it fixed in https://github.com/woocommerce/woocommerce-blocks/pull/10657/commits/606712785ef6968446b20ee32ee8aeb6f78aba22 👍